erdpy/projects/project_base.py

```diff
@@ -17,12 +18,16 @@ def __init__(self, directory: Path):
         self.path = directory.expanduser().resolve()
         self.directory = str(self.path)
 
-    def build(self, options: Union[Dict[str, Any], None] = None) -> Path:
+    def build(self, options: Union[Dict[str, Any], None] = None) -> List[Path]:
         self.options = options or dict()
         self.debug = self.options.get("debug", False)
         self._ensure_dependencies_installed()
+        if not self.options.get("no_wasm_opt"):
```

Comment:
Somehow double negation, can be extracted to a variable called `with_wasm_opt`, then used in the `if` statement (optionally, can also stay as it is).

Since `wasm_opt` is only called for Rust projects, perhaps move this logic to `project_rust`? Or perhaps enable it for C projects as well?

Reply:
Moved to `project_rust.py`.